tests/validation/tests/v3_api/common.py

```diff
@@ -430,14 +473,14 @@ def get_role_nodes(cluster, role):
 
 
 def validate_ingress(p_client, cluster, workloads, host, path,
-                     insecure_redirect=False):
+                     insecure_redirect=False, os_type=TEST_OS):
```

Comment:
I see we only have linux test case for validate ingress. Do we still need the `os_type=TEST_OS` parameters?

Reply:
Oh, we don't need this.